.gitpod.yml

```
@@ -1,3 +1,10 @@
+ports:
+    - name: Zot registry
+      description: Test OCI registry
+      port: 5000
+      # do not notify once the port is detected
+      onOpen: ignore
+      visibility: public
```

Consistent indentation, should be two spaces for YAML documents 

```suggestion
ports:
  - name: Zot registry
    description: Test OCI registry
    port: 5000
    # Do not notify once the port is detected
    onOpen: ignore
    visibility: public
```